packages/tasks/src/tasks/image-text-to-text/about.md

```diff
@@ -0,0 +1,32 @@
+## Use Cases
+
+### Visual Question Answering
```

Comment:
I would avoid to put examples of things that are covered in other task pages to avoid confusion. The 3 examples now are already covered. E.g.

- VQA https://huggingface.co/tasks/visual-question-answering
- Doc QA https://huggingface.co/tasks/document-question-answering
- Captioning https://huggingface.co/tasks/image-to-text

Reply:
I think I disagree here. one could segment humans with both image segmentation and mask detection for instance (except for zero shot part). some hugging face tasks are similar. some models are very generalistic that they can handle these tasks altogether and the paradigm is shifting more towards there anyway.

these models are similar either way, if I were an MLE who had to do captioning I'd like to try both VLMs (BTW which perform better IMO) and direct captioning models (image-to-text) which has sole purpose of captioning. I wouldn't like to keep this information from the user.

Reply:
I think it would be too repetitive. See here for instance the one in data.ts will be at the top, and then right after use case one repeats it. I wouldn't like to add it there
<img width="617" alt="Screenshot 2024-03-25 at 20 49 41" src="https://github.com/huggingface/huggingface.js/assets/53175384/06ee6f3c-0cd0-4303-ab00-3d7f844a52ca">

Reply:
My main concern is that now we have two tasks that cover the same thing, so it could end up confusing users. As an example, imagine if https://huggingface.co/tasks/text-generation use cases were Translation and Summarization, which are also their own separate tasks.

I wonder if we can use here specific applications rather than sub-tasks covered here (thinking of https://huggingface.co/tasks/text-generation#use-cases as a nice example)

Reply:
yes I agree, but I think as a specific/solid use case there's not a lot (they'd be way too specific)

Reply:
an organic way for me to infer use cases for tasks were checking out example inputs to many Spaces based on a specific task, and for VLMs it's mostly visual question answering for instance.

Reply:
I added bunch of other things